blog/_posts/2017-09-07-team_code_hoppers_blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Our journey - with rgsoc and Open Source!
+layout: post
+created_at: Thursday Sep 08 2017
+permalink: blog/2017-09-07-team_code_hoppers_blog
```

Comment:
Hi, thanks for sending your 2nd blog post PR. Your 'created at' and 'permalink' doesn't match. Please change date in your permalink and file name.

Reply:
Okay, I have implemented the changes. Do let me know if they are okay now :)